civic_to_bedpe.py

```diff
@@ -0,0 +1,126 @@
+
+
+'''
+Convert EricScript output to BEDPE format.
+
+Can be run from the commandline with:
+    python fusioncatcher_to_bedpe.py -i <input_file> -o <output_file>
+
+If no <input_file> and/or <output_file> is specified, the file will read/write
+from/to STDIN/STDOUT, allowing for piping into and out of the program.
+'''
+
+import argparse,sys,csv
+
+def map_fields(input_row, headings):
+    '''
+    Map fields from the input row to bedpe_fields
+
+    Args:
+        input_row (dict): {'heading': value} mapping for one row in input
+        headings (list): A list of all BEDPE headings to include
+    Returns:
+        dict: the return value. A single BEDPE row
+    '''
+    out_row = dict()
+
+    [chr1, start1, end1] = input_row['chromosome'], input_row['start'], input_row['stop']
+
+    out_row['chrom1'] = 'chr{num}'.format(num=chr1)
+    if(start1.isdigit() and end1.isdigit()):
+        out_row['start1'] = int(start1)
+        out_row['end1'] = int(end1)
+    else:
+        out_row['start1'] = '.'
```

Comment:
BEDPE specifies that unknown locations should be specified as `-1`

Reply:
fixing ! thanks.

Reply:
done